### PR TITLE
Make signatures with SHA-512 verify in Windows

### PR DESCRIPTION
**Pull request by netmackan**
Make output more similar to that from SignTool by:
- Filtering out CMS attributes
- Always including SPC SP OPUS Info
- Not using "<<<Obsolete>>>" string

With this changes SHA-512 is also verified correctly (issue #55).

### CODE REVIEW

File: jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public ASN1Primitive toASN1Primitive() {
         } else if (moniker != null) {
             return new DERTaggedObject(false, 1, moniker);
         } else {
-            return new DERTaggedObject(false, 2, file);
+            return new DERTaggedObject(false, 2, new SpcString(""));
```
Comment (ebourg) [inline jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java]: What about changing the `file` field instead?

**Comment by ebourg** [comment]
Thank you for the fix Markus. Is the removal of the signing time and CMS algorithm protection attributes required to validate the SHA-512 signature?

File: jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java (hunk 1 above)
Comment (netmackan) [reply jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java]: That would be better yes. Will update.

**Comment by netmackan** [comment]
It turned out that not including the extra CMS attribute was not needed. The issue seems to have been only with the spc info.

File: jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java (hunk 1 above)
Comment (netmackan) [reply jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java]: Updated in one of the last commits.

**Comment by ebourg** [comment]
Merged, thank you!

**Pull request merged**